draft-ietf-tls-dtls-rrc.md

```diff
@@ -411,7 +413,7 @@ This information can be used to identify any issues with the underlying connecti
 
 Since the DTLS 1.3 encrypted packet's record type is opaque to on-path observers, RRC messages are immune to middlebox interference when using DTLS 1.3.
 In contrast, DTLS 1.2 RRC messages that are not wrapped in the `tls12_cid` record (e.g., in the server-to-client direction if the server negotiated a zero-length CID) have the `return_routability_check` content type in plain text, making them susceptible to interference (e.g., dropping of `path_challenge` messages), which would hinder the RRC functionality altogether.
-Therefore, when using RRC in DTLS 1.2, it is recommended to enable CID in both directions.
+Therefore, when using RRC in DTLS 1.2 and middlebox interference is a concern, it is recommended to enable CID in both directions.
```

Comment:
Should we capitalize the "recommended"?

Reply:
This is in the Operational Considerations section.  I don’t know if it's expected to use RFC 2119 here.  Some people, for example, get upset if the security considerations contain normative text. 🤷